test/updater_unittest.cc

```diff
@@ -192,30 +193,28 @@ TEST_F(UpdaterTest, UpdateMetadataCallback) {
       std::chrono::system_clock::now(),
       json::object({{"f", json::string("test")}}));
   MonitoredResource resource("test_resource", {});
-  const std::string frn = "/test";
-  MetadataUpdater::ResourceMetadata metadata(
-      std::vector<std::string>({"", "test-prefix"}),
-      resource, frn, std::move(m));
+  MetadataUpdater::ResourceMetadata resource_metadata(
+      std::vector<std::string>({"", "test-prefix"}), resource, std::move(m));
   PollingMetadataUpdater updater(config, &store, "Test", 60, nullptr);
-  UpdateMetadataCallback(&updater, std::move(metadata));
-  const auto metadata_map = store.GetMetadataMap();
-  EXPECT_EQ(1, metadata_map.size());
-  EXPECT_EQ("test-type", metadata_map.at(frn).type);
-  EXPECT_EQ("test-location", metadata_map.at(frn).location);
-  EXPECT_EQ("test-version", metadata_map.at(frn).version);
-  EXPECT_EQ("test-schema-name", metadata_map.at(frn).schema_name);
-  EXPECT_EQ("{\"f\":\"test\"}", metadata_map.at(frn).metadata->ToString());
+  UpdateMetadataCallback(&updater, std::move(resource_metadata));
+  const auto metadata = store.GetMetadata();
+  EXPECT_EQ(1, metadata.size());
+  EXPECT_EQ("test-name", metadata[0].name);
+  EXPECT_EQ("test-type", metadata[0].type);
+  EXPECT_EQ("test-location", metadata[0].location);
+  EXPECT_EQ("test-version", metadata[0].version);
+  EXPECT_EQ("test-schema-name", metadata[0].schema_name);
+  EXPECT_EQ("{\"f\":\"test\"}", metadata[0].metadata->ToString());
 }
 
 TEST_F(UpdaterTest, UpdateResourceCallback) {
-  MetadataUpdater::ResourceMetadata metadata(
+  MetadataUpdater::ResourceMetadata m(
```

Comment:
Let's use `resource_metadata` for consistency with other places.

Reply:
Done